pkgs/by-name/cl/clipit/package.nix

```diff
@@ -27,16 +27,21 @@ stdenv.mkDerivation rec {
     intltoolize --copy --force --automake
   '';
 
+  # Fix build on GCC 14
+  env.NIX_CFLAGS_COMPILE = lib.optionalString stdenv.cc.isGNU "-Wno-error=implicit-function-declaration -Wno-error=incompatible-pointer-types";
```

Comment:
looks like Debian has patches to fix these:
https://salsa.debian.org/debian/clipit/-/tree/main/debian/patches?ref_type=heads

Reply:
<details><summary>gcc-14-fixes.diff</summary>

```diff
diff --git a/pkgs/by-name/cl/clipit/package.nix b/pkgs/by-name/cl/clipit/package.nix
index c19ae440579d..0ad4f7fe256d 100644
--- a/pkgs/by-name/cl/clipit/package.nix
+++ b/pkgs/by-name/cl/clipit/package.nix
@@ -1,5 +1,6 @@
 {
   fetchFromGitHub,
+  fetchpatch,
   lib,
   stdenv,
   autoreconfHook,
@@ -23,6 +24,19 @@ stdenv.mkDerivation rec {
     sha256 = "0byqz9hanwmdc7i55xszdby2iqrk93lws7hmjda2kv17g34apwl7";
   };
 
+  patches = [
+    (fetchpatch {
+      name = "gcc-14-fixes.patch";
+      url = "https://salsa.debian.org/debian/clipit/-/raw/f2e281ba0247ed8cfc1f6bae916dfac0a9c57c99/debian/patches/missing-prototypes.patch";
+      hash = "sha256-UD183IjV5BprPHQK9bhmUBKfUYgqEZ9M1cRE+AmhAPA=";
+    })
+    (fetchpatch {
+      name = "gcc-14-fixes.patch";
+      url = "https://salsa.debian.org/debian/clipit/-/raw/f2e281ba0247ed8cfc1f6bae916dfac0a9c57c99/debian/patches/incompatible-pointer-types.patch";
+      hash = "sha256-STI1fpnoPdEqu1embQcUlTG712HPbJ+LPm930P13Ixo=";
+    })
+  ];
+
   preConfigure = ''
     intltoolize --copy --force --automake
   '';

```
</details>